test/next/tsconfig.json

```diff
@@ -10,7 +10,7 @@
     "noEmit": true,
     "esModuleInterop": true,
     "module": "esnext",
-    "moduleResolution": "Node",
+    "moduleResolution": "Bundler",
```

Comment:
I didn't know about this option 👀

Reply:
This is the default of Next.js tsconfig.json nowadays. BTW "Node" means module resolution is Node 10x lol..